pg/benchmarks/pg_scope_all.rb

```diff
@@ -0,0 +1,23 @@
+require 'bundler/setup'
+require 'pg'
+require_relative 'support/benchmark_pg'
+
+# Connect to POSTGRES database with provided values or use default
```

Comment:
@tgxworld yeah, It is. I was kind of in doubt because of defaults. I'll leave it out :+1: